opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/payment/CreateSinglePaymentService.java

```diff
@@ -46,10 +48,14 @@
     @Override
     protected void doPrepareContext(DelegateExecution execution, Xs2aPisContext context) {
         context.setRedirectUriOk(
-                ContextUtil.evaluateSpelForCtx(urlsConfiguration.getPis().getWebHooks().getOk(), execution, context)
+                UriComponentsBuilder.fromHttpUrl(urlsConfiguration.getPis().getWebHooks().getOk())
+                        .buildAndExpand(ImmutableMap.of("sessionId", context.getAuthorizationSessionIdIfOpened()))
+                        .toUriString()
         );
         context.setRedirectUriNok(
-                ContextUtil.evaluateSpelForCtx(urlsConfiguration.getPis().getWebHooks().getNok(), execution, context)
+                UriComponentsBuilder.fromHttpUrl(urlsConfiguration.getPis().getWebHooks().getNok())
```

Comment:
The idea of this ticket is to replace SpEL injection with a components builder to avoid reflection access that is completely unnecessary now. This means instead of having 
```
 UriComponentsBuilder.fromHttpUrl(urlsConfiguration.getPis().getWebHooks().getOk())
                        .buildAndExpand(ImmutableMap.of("sessionId", context.getAuthorizationSessionIdIfOpened()))
```
for each case, we need to have generic builder, something like this:

```
public URI buildAndExpandQueryParameters(String urlTemplate) {
 UriComponentsBuilder.fromHttpUrl(urlTemplate)
                        .buildAndExpand(
                              ImmutableMap.of(
"sessionId", context.getAuthorizationSessionIdIfOpened(),
"redirectCode", context.getRedirectCodeIfAuthContinued(),
"wrong", context.getWrongAuthCredentials()
                         )
    );
}
```